pkg/operator/config_test.go

```diff
@@ -196,3 +184,26 @@ func TestGetMachineAPIOperatorFromImages(t *testing.T) {
 		t.Errorf("failed getMachineAPIOperatorFromImages. Expected: %s, got: %s", expectedMachineAPIOperatorImage, res)
 	}
 }
+
+func TestGetAdditionalProviderImages(t *testing.T) {
+        imagesJSONFile := "fixtures/images.json"
```

Comment:
please structure this as testCases and validate all relevant images, also see https://github.com/openshift/machine-api-operator/pull/302/files#r303407968

Reply:
Done.